Source/WebCore/dom/UserActionElementSet.h

```diff
@@ -27,10 +27,13 @@
 
 #pragma once
 
+#include "EventTarget.h"
```

Comment:
You shouldn't need this include, You can likely just forward declare `WeakPtrImplWithEventTargetData`